openapi/openapi/src/main/scala/endpoints/openapi/JsonSchemas.scala

```diff
@@ -33,6 +33,9 @@ trait JsonSchemas extends endpoints.algebra.JsonSchemas {
 
   }
 
+  def emptyRecord: DocumentedRecord =
+    DocumentedRecord(Nil)
```

Comment:
Can you do a quick check that `DocumentedRecord` usages don’t expect the list of fields to not be empty?

Reply:
Checked in the IDE and didn't find such case ✅